src/app/public/modules/vertical-tabset/vertical-tabset-adapter.service.ts

```diff
@@ -23,6 +23,10 @@ export class SkyVerticalTabsetAdapterService {
     return elementRef.nativeElement.clientWidth;
   }
 
+  public scrollToContentTop(contentEl: HTMLElement): void {
```

Comment:
DOM adapter services should receive the `ElementRef` so that our adapters can be switched out to handle other platforms (for example, a native mobile platform might have different properties on its `ElementRef`).